docs/3_2_model_exchange_api.adoc

```diff
@@ -189,60 +189,10 @@ include::../headers/fmi3FunctionTypes.h[tags=EnterEventMode]
- * `fmi3EventTrue` indicates that the given event type happens.
-
-[[fmi3EventFalse,`fmi3EventFalse`]]
- * `fmi3EventFalse` indicates that the given event type does not happen.
-
-[[fmi3EventUnknown,`fmi3EventUnknown`]]
- * `fmi3EventUnknown` indicates that the importer is unable to provide this information about this event type.
- In this case, the FMU must determine itself if this event type happens.
---
-+
-For Model Exchange, the arguments of the function have the following meaning _[For Co-Simulation, the semantics of the arguments are <<fmi3EnterEventModeCS,different>>.]_:
-+
---
-* `stepEvent` signals a <<step-event>>.
-`stepEvent` must be `fmi3EventTrue` if <<fmi3CompletedIntegratorStep>> returned with `enterEventMode = fmi3True`.
-
-* [[timeEvent,`timeEvent`]]`timeEvent` signals a <<time-event>> triggered by <<nextEventTime>> of <<fmi3UpdateDiscreteStates>>.
-
-* `stateEvent` signals a <<state-event>>, `stateEvent` is triggered by a domain change of an event indicator.
-
-* `nEventIndicators` is only defined if `stateEvent == fmi3EventTrue`.
-It contains the current number of event indicators (see <<fmi3GetNumberOfEventIndicators>>) or `0`.
-If `nEventIndicators = 0` the importer is unable to provide this information and the FMU must determine itself which event indicator has a domain change.
-
-* `eventIndicatorDomainChanges` is defined only if `stateEvent == fmi3EventTrue` and `nEventIndicators > 0`.
-`eventIndicatorDomainChanges` is an array of length `nEventIndicators` that informs the FMU which event indicator has a domain change.
-It is given in the order defined by the ordered list of XML elements <<EventIndicator>>, which is the same as in the argument <<eventIndicators>> of <<fmi3GetEventIndicators>>. +
-For `i = 0, ... nEventIndicators-1, eventIndicatorDomainChanges[i]` informs about event indicator latexmath:[{\mathbf{z}_{\mathit{i}}}]:
-+
-** `eventIndicatorDomainChanges[i] = 0` no root,
-** `eventIndicatorDomainChanges[i] = +1` indicates the direction of the domain change from negative to positive,
-** `eventIndicatorDomainChanges[i] = -1` indicates the direction of the domain change from positive to negative.
-
 +
 The importer must call <<fmi3EnterEventMode>> at a time when the signs of the event indicators signal an event according to <<localizationStateEvents>>.
-If given, the importer must ensure that `eventIndicatorDomainChanges` is consistent with the values of the event indicators. +
-If `nEventIndicators = 0` the value of `eventIndicatorDomainChanges` must be NULL.
-
---
 +
-_[An <<input-event>> can be detected by the FMU by keeping track of the calls of <<get-and-set-variable-values,`fmi3Set{VariableType}`>> in <<EventMode>>._ +
-_Please note: if all event types are given as `fmi3EventUnknown` then the functionality is the same as in FMI 2.0.]_
-
+_[An <<input-event>> can be detected by the FMU by keeping track of the calls of <<get-and-set-variable-values,`fmi3Set{VariableType}`>> in <<EventMode>>.]_
```

Comment:
Remove this part if previous rewrite is done

Reply:
Done.